x-pack/plugins/security_solution/public/management/components/endpoint_policy_link.tsx

```diff
@@ -25,20 +26,32 @@ export const EndpointPolicyLink = memo<
     backLink?: PolicyDetailsRouteState['backLink'];
   }
 >(({ policyId, backLink, children, missingPolicies = {}, ...otherProps }) => {
+  const isPolicyIdMissing = policyId === undefined || policyId === '';
```

Comment:
can you just `!policyId`? just to make sure we don't miss a non-string case here like `null` or `0`.